.github/workflows/tests.yml

```diff
@@ -7,8 +7,6 @@ on:
       - 'docs/**'
       - '*.rst'
   pull_request:
```

Comment:
Looks good. I double checked this in the docs at: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

When you omit any value here it defaults to `'**'` which matches all branch and tag names.